avet.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+@author: avetisyang
+"""
+
+
+def feature_extract(df) -> str:

```

Comment:
The function returns a pd.DataFrame object, not a string. Might be good to update that (or remove it, but updating it is better :smile: )